modules/components/tag-input.ts

```diff
@@ -802,7 +802,6 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
     private setUpOnBlurSubscriber(): void {
         this.inputForm
             .onBlur
-            .filter(() => this.dropdown && this.dropdown.isVisible)
```

Comment:
Sorry - I forgot to mention that filter needs to be there if the dropdown exists and is visible, so maybe:

```
this.dropdown && this.dropdown.isVisible ? false : true 
OR
!(this.dropdown && this.dropdown.isVisible)
```